farmerbot/internal/substrate.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import (
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	substrate "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+)
+
+// Sub is substrate client interface
+type Sub interface {
```

Comment:
Can we use a more intutive name e.g Substrate, it seems like a Subbing interface to subsitute or to subtract :D